.github/workflows/ci.yml

```diff
@@ -54,6 +54,9 @@ jobs:
       - name: Start docker stack
         run: DYNAMICTIME=false LOTIME=1000-01-01T00:00:00Z HITIME=9999-12-31T23:59:59Z docker compose up -d --wait --wait-timeout 120
 
+      - name: Test client runs without errors
+        run: DYNAMICTIME=false LOTIME=1000-01-01T00:00:00Z HITIME=9999-12-31T23:59:59Z docker compose run --rm client
+
```

Comment:
This is now run twice (see a couple of lines down). Any idea why?

Reply:
As the client is part of the profile test in the docker compose. I ran into the problem that the integration tests failed because locally they had the client, but in the pipeline not. 
I wanted to move it upwards in the pipeline. But instead of moving, I copied it.

Reply:
Originally client did not leave any data... but now it does, and the integration test depend on this data? Do we want that?

Reply:
I had my doubts about that too, the reason that I added it is that it shows current issues such as the empty string as default for gRPC instead of NULL. But I can change it as well to not include it.